README.md

```diff
@@ -27,115 +31,164 @@ repositories' pull requests by using the GitHub Pull Request Builder plugin. It
-release notes and execute a build with all packages available, by executing the
-corresponding commands from the "builds" repository scripts. Commits will be
-created for the "versions" and "open-power-host-os.github.io" repositories and
-pushed to the designated GitHub user's repository. Ideally, this should be a
-"bot" user. Pull requests for those commits can be created manually and merged
-into the main organization/user repository, if desired.
-
-Note: The default is to run those periodic builds once a week (every Wednesday
-at 11 AM).
+"start tests". Currently, tests are: building of all packages, Pylint validation
+of Python code, yamllint validation of all package metadata YAML files and rpmlint
+validation of all RPM packages specification files. These tests can be triggered
+independently with the phrases "start build", "start pylint", "start yamllint"
+and "start rpmlint" respectively.
+
+### Periodic builds process
+
+A Jenkins job will trigger periodically to check the versions of the packages
+available in OpenPOWER Host OS GitHub organization, update the corresponding
+packages' versions in the "versions" git repository, create a build with all
+packages available and create release notes, by executing the corresponding
+commands from the build scripts in the "builds" git repository. Commits will
+be created for the local "versions" and "open-power-host-os.github.io" git
+repositories and pushed to the designated GitHub user's git repository. Ideally,
+this should be a "bot" user. Pull requests for those commits can be created
+manually and merged into the main organization/user git repository, if desired.
+
+Note: The default is to execute those periodic builds once a week (every Wednesday
+at 11 AM UTC-03:00).
 
 ## Infrastructure setup
 
 If the whole infrastructure setup is desired, you should do the following steps:
-- Prepare CentOS system
-- Execute Ansible playbooks
-- Manual Jenkins steps
+- Prepare CentOS system(s)
+- Setup SSH credentials used to access upload server and GitHub from Jenkins slaves
+- Automatically setup Jenkins master and slave(s) using Ansible playbooks
+- Manually setup Jenkins via web UI
  - Create administrative jobs
- - Create slaves credentials
- - Create GitHub credentials
- - Create slave nodes
- - Create Host OS builds jobs
+ - Setup SSH credentials used to access Jenkins slaves from Jenkins master
+ - Create credentials used to access GitHub API from Jenkins
+ - Create Jenkins slaves in Jenkins web UI
+ - Create builds jobs
+
 They are described in detail below.
 
-### Prepare CentOS system
+### Prepare CentOS system(s)
+
+First, you will need to prepare at least one CentOS system. If the Jenkins
+instance will be used for testing, one system is enough; production instances
+should use more systems.  Currently, OpenPOWER Host OS has 1 Jenkins slave for
+validating the code from GitHub pull requests and 3 Jenkins slaves for builds.
```

Comment:
I agree it could be kept internally, I'm just saying that should be our doc priority ONE: document all steps to replicate *our* infrastructure. If you agree, I'll change priorities here and work on documenting these steps internally.